<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutSVAllLeft">
     <item>
      <layout class="QHBoxLayout" name="horizontalSVTopButtonsLeft">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSVTopPushButtonsLeft">
         <item>
          <widget class="QPushButton" name="pushButtonUpdateSVLeft">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Scheduled Views based on your curently entered credentials. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_34">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="SVListWidgetLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutSVBottomButtonsLeft">
       <item>
        <widget class="QPushButton" name="pushButtonSVDeleteLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Disable/Delete the currently selected Scheduled View(s).</string>
         </property>
         <property name="text">
          <string>Disable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSVBackupLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected Scheduled View(s) to file. Each SV will be written to it's own JSON file. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSVRestoreLeft">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore one or more Scheduled Views from file(s). These must be properly formatted JSON, either from a previous backup or created in accordance with the Sumo Logic API documentation. </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_40">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutSVCenterButton">
     <item>
      <spacer name="verticalSpacer_19">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSVCopyLeftToRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected scheduled view(s) from the left org to the right org.</string>
       </property>
       <property name="text">
        <string>Copy -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSVCopyRightToLeft">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Copy the selected scheduled view(s) from the right org to the left org.</string>
       </property>
       <property name="text">
        <string>&lt;- Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_20">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSVFindReplaceCopyLeftToRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
Replace-&gt;
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSVFindReplaceCopyRightToLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTip">
        <string>Coming Soon!</string>
       </property>
       <property name="text">
        <string>Find 
&lt;-Replace
Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_21">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutSVAllRight">
     <item>
      <layout class="QHBoxLayout" name="horizontalSVTopButtonsRight">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSVTopPushButtonsRight">
         <item>
          <widget class="QPushButton" name="pushButtonUpdateSVRight">
           <property name="maximumSize">
            <size>
             <width>85</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Waree</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::NoContextMenu</enum>
           </property>
           <property name="toolTip">
            <string>Update the list of Scheduled Views based on your curently entered credentials. </string>
           </property>
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="SVListWidgetRight">
       <property name="font">
        <font>
         <family>Waree</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutSVBottomButtonsRight">
       <item>
        <widget class="QPushButton" name="pushButtonSVDeleteRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Disable/Delete the currently selected Scheduled View(s).</string>
         </property>
         <property name="text">
          <string>Disable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSVBackupRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Backup the selected Scheduled View(s) to file. Each SV will be written to it's own JSON file. </string>
         </property>
         <property name="text">
          <string>Backup</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSVRestoreRight">
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Waree</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Restore one or more Scheduled Views from file(s). These must be properly formatted JSON, either from a previous backup or created in accordance with the Sumo Logic API documentation. </string>
         </property>
         <property name="text">
          <string>Restore</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_39">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
